.flowconfig

```diff
@@ -15,6 +15,7 @@ module.name_mapper='fos-jsrouting/router' -> 'empty/object'
 .*/node_modules/jss/.*
 .*/node_modules/npmconf/.*
 .*/node_modules/stylelint/.*
+.*/node_modules/resolve/test/resolver/malformed_package_json/.*
```

Comment:
The `resolve` package is publish its package with tests. Which have unparseable files in it. Sadly a pull request to avoid this:

https://github.com/browserify/resolve/pull/270
https://github.com/browserify/resolve/pull/269

Was not accepted so we need to add this to our flow config and users using flow to there flow config.

Reply:
🙈